.circleci/config.yml

```diff
@@ -240,18 +240,30 @@ jobs:
     steps:
       - checkout_and_fetch_tags
       - restore_node_modules
+      - add_ssh_keys:
```

Comment:
This can't be done in circle config?

Reply:
Nope it needs to be added manually (tested without it yesterday and it defaulted to a non-writable key)